app/app.go

```diff
@@ -855,7 +855,7 @@ func (app *App) RegisterTendermintService(clientCtx client.Context) {
 
 // RegisterSwaggerAPI registers swagger route with API Server
 func RegisterSwaggerAPI(ctx client.Context, rtr *mux.Router) {
-	statikFS, err := fs.New()
+	statikFS, err := fs.NewWithNamespace("cronos")
```

Comment:
used a different namespace to prevent potential conflict with the one used by cosmos-sdk's simd.